charts/karavi-observability/Chart.yaml

```diff
@@ -8,3 +8,4 @@ dependencies:
 - name: cert-manager
   version: 1.10.0
   repository: https://charts.jetstack.io
+  condition: cert-manager.enabled
```

Comment:
cert-manager is mandatory required for Observability.
'condition' is not necessary as it has to be enabled.

Reply:
Thanks, Yian Zong, for reviewing the code. We have created csm helm chart to install multiple drivers and modules and cert-manager is a dependency for multiple modules, so we added it as dependency in the csm chart to install it only for one time and disabled in all the modules in the csm chart, so we required this condition to be added in all the modules dependent on cert-manager. By default, we enabled the cert-manager in observability chart so it will not change the existing functionality of the chart.

Reply:
@chaganti-rajitha Thank you for clarifying. Make sense to me.